pod/perlsub.pod

```diff
@@ -1545,13 +1545,14 @@ will allow calling myref() as
     myref *glob
 
 and the first argument of myref() will be a reference to
-a scalar, an array, a hash, a code, or a glob.
+a scalar, an array, a hash, a subroutine, or a glob.
 
 Unbackslashed prototype characters have special meanings.  Any
 unbackslashed C<@> or C<%> eats all remaining arguments, and forces
 list context.  An argument represented by C<$> forces scalar context.  An
 C<&> requires an anonymous subroutine, which, if passed as the first
-argument, does not require the C<sub> keyword or a subsequent comma.
+argument, may look like a bare block:  It does not require the C<sub> keyword
```

Comment:
Did you mean to write a colon here? Maybe s/block:/block./

Reply:
I did, but I'm not opposed to changing it to a `.` either.